Web.md

```diff
@@ -14,6 +14,12 @@ than if the module was JavaScript.
 More concretely, the following is a list of points of contact between WebAssembly
 and the rest of the Web platform that have been considered:
 
+## The `Wasm` object
+
+WebAssembly is exposed to the Web through a `Wasm` object, whose API is designed
```

Comment:
This isn't the only way Wasm is exposed to the Web (or even JS).  With ES6 module integration, there'd also be <script type=module> as well import from ES6.  Also, with Web API integration, one should be able to pass the URL of a .wasm to a Worker constructor.